test/parallel/test-process-should-abort-on-uncaught.js

```diff
@@ -2,7 +2,7 @@
 'use strict';
 const common = require('../common');
 
-// This should make the process not crash even though the flag was passed.
+// This should make the process does not crash even though the flag was passed.
```

Comment:
I think the change might've turned this into a typo? I think the previous version was just referring to the next line in which case it was grammatically correct.

Reply:
lol yes i just noticed myself :smile: